test/lib/jdk/test/lib/Asserts.java

```diff
@@ -446,8 +445,7 @@ public static void assertNotEquals(Object lhs, Object rhs) {
     public static void assertNotEquals(Object lhs, Object rhs, String msg) {
         if ((lhs == rhs) || (lhs != null && lhs.equals(rhs))) {
             msg = Objects.toString(msg, "assertNotEquals")
-                    + ": expected " + Objects.toString(lhs)
-                    + " to not equal " + Objects.toString(rhs);
+                    + ": expected not equals but was " + Objects.toString(lhs);
```

Comment:
Same comment as above `assertNotEqualsByteArray(byte[] lhs, byte[] rhs, String msg)`

Reply:
This might hide problems with the implementation of `equals`

Reply:
But this would be very uncommon and once it happens the developer would find out the problem, right?

Maybe I can use `Objects.toString(rhs)` in the message. `lhs` (as the expected value) is more likely to be a constant.